### PR TITLE
maptext barsigns

### PR DESCRIPTION
**Pull request by JTGSZ**
The fourth iteration of the custom maptext barsign testcase, this time it has a shitty assc list queue system for the custom side, and a menu for pre-defined barsigns in one package. It also has a low amount of filter effects possible.

argh
argh1

### CODE REVIEW

**Comment by SonixApache** [comment]
log the text

**Comment by PrimeDSS13** [comment]
This is pretty cool. Can we see some example signs?

File: code/game/objects/structures/barsign.dm [hunk 1]
```diff
@@ -29,59 +30,251 @@
 	icon = "maltesefalcon"
 	desc = "Play it again, sam."
 
+/obj/effect/overlay/kustom_barsign
```
Comment (ShiftyRail) [inline code/game/objects/structures/barsign.dm]: >kustom

Played too much Ork in /vg/ 40k I think

**Comment by JTGSZ** [comment]
Closing this because Imma do some more stuff/maybe rework it

**Comment by SonixApache** [comment]
the sounds are going to be fucking horrible but we'll see how bad it is

**Comment by jknpj** [comment]
I welcome our WGW barsigns.

**Comment by DamianX** [comment]
pretty horrible
https://s1.webmshare.com/EGa1x.webm

**Comment by ShiftyRail** [comment]
Oh yeah

**Pull request merged**